src/main/webapp/app/tasks/services/local-task.service.ts

```diff
@@ -40,4 +50,8 @@ export class LocalTaskService implements TaskService {
   private writeTasks(tasks: Task[]): void {
     localStorage.setItem(LocalTaskService.STORAGE_KEY, JSON.stringify(tasks));
   }
+
+  private includesAllTermsInTaskName(terms: string[]): (task: Task) => boolean {
+    return (task: Task) => task.name && terms.every((term: string) => task.name.includes(term));
```

Comment:
is it on purpose that you made the search case-sensitive?

Reply:
Actually, yes.
On the other hand there were not any requirements for case-sensitivity in search.